_pages/dev/lazy-loading-guide.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Lazy Loading Guide
+---
+
+Lazy loading, also known as code splitting, lets you divide your JavaScript code into multiple chunks. The result is that you do not have to load all of the chunks when a user first accesses your storefront. Instead, chunks are only loaded when they are needed.
+
+Such an approach can substantially improve "Time To Interactive", especially in the case of complex web applications being accessed by low-end mobile devices.
+
+## Spartacus Approach to Lazy Loading
+
+Spartacus is mostly CMS driven, which means Spartacus does not use much of the built-in, route-based lazy loading that is offered out-of-the-box by Angular. Code splitting has to be done at app build time, although Spartacus cannot know what components or features a specific route can contain before actually loading them.
```

Comment:
> has to be done at app build time

has to be done at application build time

Reply:
👍